osf/models/collection.py

```diff
@@ -91,6 +107,9 @@ def save(self, *args, **kwargs):
         if first_save:
             # Set defaults for M2M
             self.collected_types = ContentType.objects.filter(app_label='osf', model__in=['abstractnode', 'collection'])
+            # Set up initial permissions
+            self.update_group_permissions()
```

Comment:
@mfraezz Is there a reason this wasn't put on `GuardianMixin`?

Reply:
Not as far as I recall. I'll likely move it there whenever we use `GuardianMixin` for something else.

Reply:
OK cool. @chennan47 will be using `GuardianMixin` for preprints as part of the Divorce. Nan, feel free to move this to `GuardianMixin` in your branch.